syntax = 'proto3';
import "google/protobuf/timestamp.proto";
service Auth {
  rpc Register(RegisterRequest) returns (RegisterResponse);
  rpc Login(LoginRequest) returns (LoginResponse);
}
message RegisterRequest {
  string login = 1;
  string password = 2;
}
message SendResponse {
  bool success = 1;
  string error = 2;
  string id = 3;
  google.protobuf.Timestamp server_time = 4;
}
message RegisterResponse {
  bool success = 1;
  string error = 2;
}
message LoginRequest {
  string login = 1;
  string password = 2;
}
message LoginResponse {
  bool success = 1;
  string token = 2;
  string error = 3;
}